crowbar_framework/app/models/backup.rb

```diff
@@ -23,6 +23,9 @@ class Backup < ActiveRecord::Base
   before_validation :save_or_create_archive, on: :create
   after_validation :delete_archive, unless: -> { errors.empty? }
   after_destroy :delete_archive
+  after_create do
```

Comment:
I am a bit lost here. I don't have any idea how to achieve this.
* add the `data.rmtree` to more callbacks (wouldn't help)
* use `ObjectSpace.define_finalizer` (discouraging) see e.g. [here](https://www.ruby-forum.com/topic/17155)

how can we fix this?